### PR TITLE
Expose list value typings [8.12]

### PR DESCRIPTION
**Pull request by r0b1n**

### CODE REVIEW

File: content/apidocs-mxsdk/apidocs/client-apis-for-pluggable-widgets.md [hunk 1]
```diff
@@ -213,36 +213,94 @@ export interface ListValue {
...
+
+`ListAttributeValue` represent an [attribute property](property-types-pluggable-widgets#attribute) that is linked to a data source.
+This allows the client component to access attribute values on individual items from a `ListValue`. 
+`ListAttributeValue` is a function and its definition is as follows:
+
+```ts
+export type ListAttributeValue = (item: ObjectItem) => EditableValue<AttributeValue>;
+```
+
+In order to work with the attribute value of a particular item of a `ListValue` first an instance of `ActionValue` should be obtained by calling `ListAttributeValue` with the item. See an example below.
```
Comment (mlaponder) [inline content/apidocs-mxsdk/apidocs/client-apis-for-pluggable-widgets.md:260]: ActionValue -> EditableValue

File: content/apidocs-mxsdk/apidocs/client-apis-for-pluggable-widgets.md [hunk 2]
```diff
@@ -213,36 +213,94 @@ export interface ListValue {
...
 }
 ```
 
-Note that `myStringAttribute` is a function that returns an [`EditableValue`](#editable-value). In order to get the value, an item from `ListValue` should be supplied. See this code for an example:
+The following code sample shows how to get an `EditableValue` that represents an value of an attribute of the first element from the `myDataSource`.
 
+```ts
+const attributeValue = this.props.myDataSource.myAttributeOnDatasource(this.props.myDataSource.item[0]);
+```
+
```
Comment (mlaponder) [inline content/apidocs-mxsdk/apidocs/client-apis-for-pluggable-widgets.md:277]: Maybe it is worth mentioning the value is read only

File: content/apidocs-mxsdk/apidocs/client-apis-for-pluggable-widgets.md [hunk 3]
```diff
@@ -213,36 +213,94 @@ export interface ListValue {
...
+Note: in this code sample checks of status of `myDataSource` and availability of items are omited for simplicity. See [EditableValue section](#editablevalue) for more information about usage of `EditableValue`.
+
+
+### 4.9 ListWidgetValue {#listwidgetvalue}
+`ListWidgetValue` represents an [widget property](property-types-pluggable-widgets#widgets) that is linked to a data source.
```
Comment (mlaponder) [inline content/apidocs-mxsdk/apidocs/client-apis-for-pluggable-widgets.md:282]: an -> a

File: content/apidocs-mxsdk/apidocs/client-apis-for-pluggable-widgets.md (hunk 2 above)
Comment (ConnorLand) [reply content/apidocs-mxsdk/apidocs/client-apis-for-pluggable-widgets.md:277]: Done

File: content/apidocs-mxsdk/apidocs/client-apis-for-pluggable-widgets.md (hunk 1 above)
Comment (ConnorLand) [reply content/apidocs-mxsdk/apidocs/client-apis-for-pluggable-widgets.md:260]: Changed. With technical changes like this, you can change them directly if you wish.

Comment (r0b1n) [reply content/apidocs-mxsdk/apidocs/client-apis-for-pluggable-widgets.md]: @ConnorLand I was going to push the changes regarding Marco's comments but stuck with some other tasks for some time. Thank you for fixing them 😎 !

File: content/apidocs-mxsdk/apidocs/client-apis-for-pluggable-widgets.md [hunk 4]
```diff
@@ -213,36 +213,94 @@ export interface ListValue {
...
+const actionOnFirstItem = this.props.myDataSource.myListAction(this.props.myDataSource.item[0]);
+
+actionOnFirstItem.execute();
 ```
 
-Similarly a property of type `attribute` with a linked datasource is represented in the client component as follows:
+Note: in this code sample checks of status of `myDataSource` and availability of items are omited for simplicity. See [ActionValue section](#actionvalue) for more information about usage of `ActionValue`.
+
+
+### 4.8 ListAttributeValue {#listattributevalue}
```
Comment (mijay) [inline content/apidocs-mxsdk/apidocs/client-apis-for-pluggable-widgets.md:250]: I think there are a few references to "old-style" types in https://docs.mendix.com/apidocs-mxsdk/apidocs/property-types-pluggable-widgets#attribute

**Pull request merged**